### PR TITLE
Cleanup + Compatibility with Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @grosser

### CODE REVIEW

File: lib/soft_deletion/core.rb [hunk 1]
```diff
@@ -80,15 +71,15 @@ def mark_as_undeleted
     end
 
     def soft_delete!
-      _run_soft_delete { save! }
+      result = _run_soft_delete { save! }
```
Comment (grosser) [inline lib/soft_deletion/core.rb]: no need to assign result ?

File: soft_deletion.gemspec [hunk 2]
```diff
@@ -8,11 +8,11 @@ Gem::Specification.new name, SoftDeletion::VERSION do |s|
   s.homepage = "https://github.com/grosser/#{name}"
   s.files = `git ls-files lib Readme.md`.split("\n")
   s.license = "MIT"
-  s.add_runtime_dependency 'activerecord', '>= 3.2.0', '< 4.3.0'
```
Comment (grosser) [inline soft_deletion.gemspec]: add ruby >= 2.0 since 1.9 is no longer tested

**Comment by grosser** [comment]
:+1: :heart: so much cleaner :)

**Comment by grosser** [comment]
I'll do the cleanup + ruby version and then bump

File: lib/soft_deletion/core.rb (hunk 1 above)
Comment (pschambacher) [inline lib/soft_deletion/core.rb]: This was a mistake :-/

**Comment by grosser** [comment]
0.7.0

**Pull request merged**